src/simd/simd.cpp

```diff
@@ -97,71 +59,33 @@ setup_simd() {
 #ifndef ENABLE_AVX512
     }
 #else
-        L2SqrSIMD16Ext = L2SqrSIMD16ExtAVX512;
-        InnerProductSIMD16Ext = InnerProductSIMD16ExtAVX512;
     }
     ret.dist_support_avx512f = true;
     ret.dist_support_avx512dq = true;
     ret.dist_support_avx512bw = true;
     ret.dist_support_avx512vl = true;
 #endif
-
     return ret;
 }
 
 DistanceFunc
 GetInnerProductDistanceFunc(size_t dim) {
```

Comment:
Can we remove the functions? Just use `vsag::InnerProductDistance` replace it

Reply:
too much changes, in this pr ,only split the function, but not change usage